tests/ml/pytorch/test_pytorch_model_upload_pytest.py

```diff
@@ -103,17 +103,27 @@ def download_model_and_start_deployment(tmp_dir, quantize, model_id, task):
 
 
 class TestPytorchModel:
+    def __init__(self):
+        # quantization does not work on ARM processors
```

Comment:
If you think the situation is better in PyTorch 2.0 please add a TODO to revisit this when we upgrade.